utils/cryptoTransaction.js

```diff
@@ -0,0 +1,29 @@
+function isBalanceSufficient (senderCoins, reveicerCoins) {
+  if (senderCoins.copper >= reveicerCoins.copper && senderCoins.silver >= reveicerCoins.silver && senderCoins.gold >= reveicerCoins.gold) {
+    return true
+  }
+  return false
+}
+
+function updateBalance (payer, receiver, requestCoins) {
```

Comment:
Let's change this function and the function on line no 1 to arrow functions, because the functions on line no 22 & 24 are arrow functions, so let's follow one format throughout.

Reply:
I thought normal function would be easy to understand, I will change it to arrow function